airflow/migrations/versions/0069_2_0_0_add_scheduling_decision_to_dagrun_and_.py

```diff
@@ -48,7 +48,9 @@ def upgrade():
 
     with op.batch_alter_table("dag_run", schema=None) as batch_op:
         batch_op.add_column(sa.Column("last_scheduling_decision", TIMESTAMP, nullable=True))
-        batch_op.create_index("idx_last_scheduling_decision", ["last_scheduling_decision"], unique=False)
+        # Earlier we had here an index created on the last_scheduling_decision column.
```

Comment:
Do we need to delete it from here? Just the new migrations should be enough. 

Referring the example - of `idx_not_null_dag_id_run_id` Got created in 0088_2_2_0_improve_mssql_compatibility.py and deleted in `0093_2_2_0_taskinstance_keyed_to_dagrun.py`

Reply:
We we wont cherry pick this change in 2.0.0 right? So I guess we cannot do anything for 2.0.0, I agree the new migration should be enough

Reply:
okay removed those changes 👍🏽